docs/source/release-notes.html.md

```diff
@@ -7,6 +7,27 @@ weight: 8
 
 If you have any questions or comments about these notes, please contact DfE via Slack or email.
 
+## 2nd June 2023
+
+Providers can now make calls to API 3.0.0.0. in the production environment.
```

Comment:
```suggestion
Providers can now make calls to API 3.0.0 in the production environment.
```
I know this came from the release note but think it should be with less zeros :)